intro-to-R-tidyverse/03-intro_to_r_tidyverse_exercise.Rmd

```diff
@@ -0,0 +1,160 @@
+                                                                                                                                                                                                                                                                                                                                                                                ---
+title: "Exercise: Clean Metadata and Prepare Gene Expression Matrix for Downstream Analysis"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+The goal of these exercises is to help you get familiar with using tidyverse functions to manipulate data. 
+
+Here, 
+TODO: Determine suitable experiment and write about it
+
+### Setup 
+Before we start the exercises, we need to create the necessary directories and loading the necessary libraries.
+
+#### Creating necessary directories
+We need to create a directory to store the results of the exercises. We have done this for you. 
+Run the code chunk below and take a look at your files pane. You should see a directory `exercise-results` created. 
+
+```{r}
+#creating required directories as needed
+if (!dir.exists("excercise-results")) {
+  # ... create a 'results' directory
+  dir.create("exercise-results")
+}
+```
+
+#### Loading libraries
+For these exercises we will need the `tidyverse` library. Go ahead and load the library in the code chunk below. Don't forget to run the code chunk once you write the command! 
+
+```{r}
+#load tidyverse library
+
+```
+
+#### Importing Files
+Let's import all the files we will need for this exercise. 
+The metadata is present in `exercise-data/metadata_GSE49243.tsv` and gene expression matrix is in `exercise-data/GSE49243.tsv`
+Go ahead and read the metadata and gene expression matrix into variables called `metadata` and `gene_expression`
+
+```{r}
+# Reading Metadata
```

Comment:
Here you've started capitalizing your comments, I don't care if you capitalize them or choose not too, but probably stick with the same for all?